### PR TITLE
test: add unit tests for exec.go file in pkg/utils folder

### LINKED ISSUE #752: Add unit tests for RandStringBytesMask and GetRunID in pkg/utils/stringutils
## Description

The functions `RandStringBytesMask` and `GetRunID` in `pkg/utils/stringutils/string.go` currently do not have unit test coverage.

### Proposed Solution

- Write table-driven tests for `RandStringBytesMask`, including edge case for `n = 0`
- Write basic validation test for `GetRunID` (check length and character set)
- Improve code coverage for the file

### Why it's needed

This helps ensure the reliability of utility functions used across chaos experiments and increases test coverage.

### Additional context

Prepared test locally and verified coverage using:
```bash
go test -cover ./pkg/utils/stringutils

### PR DESCRIPTION
**Pull request by Prakhar-Shankar**
**What this PR does / why we need it**:

This PR adds unit tests for two utility functions in the `exec.go` file inside the `utils` package:
- `SetExecCommandAttributes`
- `checkPodStatus`

These are pure functions and testable without external dependencies.  
This contribution is part of a larger effort to improve test coverage across the `utils` package.

> ⚠️ Note: This PR provides **partial coverage** (currently ~33%) since the `Exec()` function depends on Kubernetes APIs and remote SPDY execution, which would require more advanced mocking or integration setup. That work can be done in a follow-up PR.

---

Fixes #752 

---

**Special notes for your reviewer**:
- Tests are written with table-driven and edge case validation
- Maintainers are welcome to suggest how to approach testing `Exec()` in a future PR (e.g., mocking clients or using `fake.Clientset`)
- This aligns with LitmusChaos goals of improving code quality and test coverage

---

**Checklist:**
- [x] Fixes #752 
- [x] PR message has documentation-related information
- [ ] Labelled this PR & related issue with `breaking-changes` tag *(Not applicable)*
- [ ] PR message has breaking changes related information *(N/A)*
- [ ] Labelled this PR & related issue with `requires-upgrade` tag *(N/A)*
- [ ] PR message has upgrade related information *(N/A)*
- [x] Commit has unit tests
- [ ] Commit has integration tests *(N/A)*
- [ ] E2E run Required for the changes *(N/A)*

### CODE REVIEW

File: pkg/utils/exec/exec_test.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	p := &PodDetails{}
+	SetExecCommandAttributes(p, "test-pod", "test-container", "default")
+
+	if p.PodName != "test-pod" || p.ContainerName != "test-container" || p.Namespace != "default" {
+		t.Errorf("SetExecCommandAttributes failed to set values properly")
+	}
+}
+
+func TestCheckPodStatus(t *testing.T) {
+	// I am checking these three conditions -
```
Comment (neelanjan00) [inline pkg/utils/exec/exec_test.go:20]: We can remove this line

File: pkg/utils/exec/exec_test.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+func TestSetExecCommandAttributes(t *testing.T) {
+	p := &PodDetails{}
+	SetExecCommandAttributes(p, "test-pod", "test-container", "default")
+
+	if p.PodName != "test-pod" || p.ContainerName != "test-container" || p.Namespace != "default" {
+		t.Errorf("SetExecCommandAttributes failed to set values properly")
+	}
+}
+
+func TestCheckPodStatus(t *testing.T) {
```
Comment (neelanjan00) [inline pkg/utils/exec/exec_test.go:19]: If possible can we re-write the cases in a tabular format?

**Review by neelanjan00 — CHANGES_REQUESTED** [review]
